www/front_src/src/styles/partials/_side-nav.scss

```diff
@@ -395,7 +395,7 @@ a {
       font-size: 0.8rem;
       background-color: $white-color;
       display: block;
-      padding: 3px 10px 3px 47px;
+      padding: 3px 10px 3px 56px;

```

Comment:
modification from another PR not yet validated by the QA -> to remove from this PR